server/src/main/java/org/elasticsearch/index/codec/PerFieldFormatSupplier.java

```diff
@@ -31,19 +34,33 @@
  * vectors.
  */
 public class PerFieldFormatSupplier {
+    public static final FeatureFlag USE_LUCENE101_POSTINGS_FORMAT = new FeatureFlag("use_lucene101_postings_format");
```

Comment:
@jordan-powers @martijnvg  Lucene 10.3 (still in development) will bring a new postings format. Is the intention that this feature flag to select the "latest" codec? Or is it really specific to 10.1?  I ask only because we need to determine if a `use_lucene101_postings_format` is warranted or maybe make this flag more general?

Reply:
The intent is the use the latest default lucene postings format. Unfortunately the name doesn't reflect that, but we want it to use the latest default postings format.